DataDefinitions/ModuleDefinitions.cs

```diff
@@ -900,11 +900,6 @@ public class ModuleDefinitions
                 {128788705, new Module(128788705, "Hpt_ATDumbfireMissile_Turret_Large", 1624, "AX Missile Rack", 3, "A", 3866338, Module.ModuleMount.Turreted) },
                 {128788702, new Module(128788702, "Hpt_ATMultiCannon_Fixed_Large", 1625, "AX Multi-Cannon", 3, "C", 1126044, Module.ModuleMount.Fixed) },
                 {128793060, new Module(128793060, "Hpt_ATMultiCannon_Turret_Large", 1626, "AX Multi-Cannon", 3, "E", 3642224, Module.ModuleMount.Turreted) },
-                {128785621, new Module(128785621, "Type9_Military_Armour_Grade1", 1627, "Lightweight Alloy", 1, "I", 0, 128785619) },
-                {128785622, new Module(128785622, "Type9_Military_Armour_Grade2", 1628, "Reinforced Alloy", 1, "I", 48654583, 128785619) },
-                {128785623, new Module(128785623, "Type9_Military_Armour_Grade3", 1629, "Military Grade Composite", 1, "I", 109472813, 128785619) },
-                {128785624, new Module(128785624, "Type9_Military_Armour_Mirrored", 1630, "Mirrored Surface Composite", 1, "I", 258720748, 128785619) },
-                {128785625, new Module(128785625, "Type9_Military_Armour_Reactive", 1631, "Reactive Surface Composite", 1, "I", 283830162, 128785619) },
```

Comment:
I think we'd prefer to delete the other set - this set is in the correct order (sorted by EDDB ID)

Reply:
Better now. Thanks.